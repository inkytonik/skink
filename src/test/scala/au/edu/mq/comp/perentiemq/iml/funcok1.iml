def funcok1(x, y): (o1, o2) {
    a = 1;
    o1 = x + y;
    o2 = 100;
}
