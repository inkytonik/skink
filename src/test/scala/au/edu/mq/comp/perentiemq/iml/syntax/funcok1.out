Program (
    Vector (
        FunDef (
            "funcok1",
            Vector (Arg ("x"), Arg ("y")),
            Vector (Arg ("o1"), Arg ("o2")),
            Block (
                Vector (
                    Assign (Vector (Var ("a")), Vector (IntLit (1))),
                    Assign (
                        Vector (Var ("o1")),
                        Vector (Add (ELoad (Var ("x")), ELoad (Var ("y"))))),
                    Assign (Vector (Var ("o2")), Vector (IntLit (100))))))))
def funcok1(x, y): (o1, o2) {
    a = 1;
    o1 = x + y;
    o2 = 100;
}

